lib/matplotlib/widgets.py

```diff
@@ -3443,6 +3457,66 @@ def __init__(self, ax, onselect, useblit=False,
         self.grab_range = grab_range
 
         self.set_visible(True)
+        self._draw_box = draw_box
+        self._box = None
+
+    def _get_bbox(self):
+        return self._selection_artist.get_bbox()
+
+    def _add_box(self):
+        self._box = RectangleSelector(self.ax,
+                                      onselect=self._scale_polygon,
+                                      useblit=self.useblit,
+                                      interactive=True)
```

Comment:
Change it on move, instead of onselect:
```suggestion
        self._box = RectangleSelector(self.ax,
                                      onselect=lambda *args, **kwargs: None,
                                      useblit=self.useblit,
                                      interactive=True)
        self._box.connect_event('motion_notify_event', self._scale_polygon)
```
`self._scale_polygon` will need to be changed accordingly.

Reply:
Ah amazing, I was wondering how to do this!